### ISSUE TITLE
win 11

### ISSUE THREAD

**Issue by blindfaced** [helge17/tuxguitar#23]
doesn't work with win 11 on my computer, had to install for win 8

when launched 2 errors

cannot invoke "org.herac.tuxguitar.JackClient.addPortListener
(org.herac.tuxguitar.jack.JackPortRegisterListener)" 
because "this.JackClient) is null

C:\ProgramFiles\tuxguitar-2023-04-30-snapshot\lib\tuxguitar-jack-jni.dll: 
Can't find dependent libraries

**Comment by guiv42** [comment]
Tried with swt portable version (not the installer) on a win11 machine, and reproduced the same issue.
As a workaround (and until a clean solution is found) you cant try to delete manually the following files:
- share/plugins/tuxguitar-jack.jar
- share/plugins/tuxguitar-jack-ui.jar
- lib/tuxguitar-jack-jni.dll

after that it worked for me

**Comment by helge17** [comment]
In the TuxGuitar default settings for Windows, the Jack plugin was enabled by default. The errors should disappear if you disable the "Jack Audio Connection Kit Plugin" under "Tools - Plugins". If you want to use the Jack plugin, you need to download and install Jack from https://jackaudio.org/downloads/.

Commit 9b715d8 (binary release 2023-05-01) disables the Jack plugin by default on Windows, so users who do not need Jack will not get an error message if Jack is not installed. You may need to remove the TuxGuitar configuration folder AppData\Roaming\tuxguitar to get rid of the old setting.

**Comment by guiv42** [comment]
Just one question: it's a bit weird, on page https://github.com/helge17/tuxguitar/tags, tag 2023-05-01 is dated from "yesterday" (?!) and the associated commit shown is f5fafce1e8b591f898e3c73f1b983857d555e7c9, not the last one 9b715d831b05629f6b6100908813dd6564576a09. I assume it's just a display error?
![Screenshot_20230501_214956](https://user-images.githubusercontent.com/129443524/235519408-1dff8fa2-87f4-449f-863c-c35b41c5d840.png)

By the way, for some obscure reason when I tested with Win11 the app was automatically closed after the error was displayed (i.e. not possible to modify settings from the app menus). So, disabling Jack by default definitely seems to be the best solution

**Comment by helge17** [comment]
I just uploaded version 2023-05-01 three hours ago, but I built the packages and started the upload before committing 9b715d8, maybe that caused the confusion. I think I need to start the upload after the commit next time, sorry....

On my Win 10, TuxGuitar shows the errors, but after I clicked them away, TuxGuitar starts normally.

Anyway, the new release should fix the error (after deleting AppData\Roaming\tuxguitar).
